7.1 Snapshot.md

```diff
@@ -0,0 +1,53 @@
+# 7.1 Snapshot
+
+## 7.1.1 Introduction
+This section defines the *Snapshot* protocol.
+
+
+## 7.1.2 Dependencies
+The *Snapshotting* protocol depends on:
+
++ Timestamps: Core module to enable pruning.
++ Tip Selection Algorithm: No messages can be referenced for the part of the Tangle that is being pruned.
++ UTXO: Transactions whose outputs are unspent, remain after the snapshot.
++ Branches: We require that the at the point of pruning only master branch and rejected branch exist.
++ Markers: We require that the part of the Tangle being pruned is not actively dependent on Markers any more. 
++ Approval weight: We require that the approval weight is concluded at a time much before messages reach the pruning age.
++ Voting: We require that any voting is concluded much before messages reach the pruning age. 
++ Data flow: A weak past cone check is applied.
+
+
+## 7.1.3 Proof of Inclusion and Pruning
+
+In order to keep the ledger concise, data that is sufficiently old can be removed. We call this process pruning. Thus, the IOTA 2.0 protocol for a standard node does not attempt to keep all information back until the Genesis. Rather the protocol effectively becomes a mechanism to prove whether a certain message was, at some point in the past, included in the ledger. This can be done e.g. through a Merkle tree proof. 
+
+Transactions are payloads of value messages, that cannot be pruned. Specifically when a transaction contains an unspent output, see [5.1. UTXO](./5.1%20UTXO.md), the transaction shall be kept when pruning the ledger. An optimization of this process is possible by reducing the message to only keeping the information about the unspent outputs, however the implementation will be more complicated, and thus is not considered for this first iteration of the protocol.
+
+## 7.1.5 Pruning dependencies
+
+We require several conditions to be fulfilled in order to allow for a smooth pruning process. Pruning is done when messages reach the age `DELTA_PRUNE`. 
+
+Since messages cannot attach to a message whose timestamp has an age difference of more than `DELTA`, see [4.3.4 R-URTS](./4.3%20Tip%20Selection%20Algorithm.md/#4.3.4%20R-URTS), and the timestamp precision is enforced through voting, see [4.2.3 General Timestamp rules](./4.2%20Timestamps.md/#4.2.3%20General%20Timestamp%20rules). 
```

Comment:
This phrase has no Conclusion, you should say "Since A and B  happens, then C happens", but you just say "Since A and B happens". It may be that the "Since" is a typo, but it requires some change.

Reply:
typo. corrected it.